scripts/travis-test.sh

```diff
@@ -27,15 +27,16 @@ log_and_run() {
 }
 
 run_tests() {
-  (cd ${ROOT_DIR} && log_and_run test/run-tests.py ${RUN_TEST_ARGS} $* --timeout=10)
+  (cd ${ROOT_DIR} && log_and_run test/run-tests.py --bindir ${BINDIR} $* --timeout=10) && true
+  (log_and_run ${BINDIR}/wabt-unittests) && true
```

Comment:
Not sure the right way to do this stuff in shell script. But the errexit was making it blow up without running the other tests, which I didn't want. And I thought the subshell would override that, but it doesn't seem to do it without `(...) && true`

Maybe better to just disable errexit here?